diff -up gtk-4.0.0/gdk/gdkvulkancontext.c.omv~ gtk-4.0.0/gdk/gdkvulkancontext.c
--- gtk-4.0.0/gdk/gdkvulkancontext.c.omv~	2020-12-17 00:45:49.171532660 +0100
+++ gtk-4.0.0/gdk/gdkvulkancontext.c	2020-12-17 00:46:01.290674378 +0100
@@ -210,8 +210,10 @@ gdk_vulkan_strerror (VkResult result)
       return "An unknown error has occurred; either the application has provided invalid input, or an implementation failure has occurred.";
 #endif
 #if VK_HEADER_VERSION >= 135
+#ifdef VK_ERROR_INCOMPATIBLE_VERSION_KHR
     case VK_ERROR_INCOMPATIBLE_VERSION_KHR:
       return "Acceleration structure serialized with version as the version information is not compatible with device.";
+#endif
     case VK_THREAD_IDLE_KHR:
       return "A deferred operation is not complete but there is currently no work for this thread to do at the time of this call.";
     case VK_THREAD_DONE_KHR:
